apps/src/code-studio/components/pairing/StudentSelector.jsx

```diff
@@ -47,29 +69,48 @@ export default class StudentSelector extends React.Component {
     } else if (this.props.students.length === 0) {
       return <span>{i18n.noStudentsInSection()}</span>;
     }
+    const {students} = this.props;
+    const {selectedStudentIds} = this.state;
```

Comment:
If you bump these two up to the start of the render function, you can take advantage of them in lines 67 and 69.